app/src/main/res/drawable/pocket_vector.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<vector android:autoMirrored="true" android:height="22dp"
```

Comment:
Might want to run a format here. I am not used to seeing multiple android:X properties on the same line. and xmls:android at the bottom.

Reply:
Done.